grails-app/controllers/org/pih/warehouse/inventory/StockMovementController.groovy

```diff
@@ -66,6 +66,10 @@ class StockMovementController {
         render(view: "/common/react", params: params)
     }
 
+    def fullOutboundImport() {
```

Comment:
Do we have to have a "full" keyword everywhere?

Reply:
Yeah I was going to say we really should keep things simple. Try not to use PIH terminology if there's a way to make it simple. Imagine some old Greek guys having a conversation and then distill their response. 

> Socrates: What does it do? 
> Kacperian: It provides a way to import a stock movement including all of its dependent objects like requests, picklists, etc. 

:sparkles: 
```
importStockMovement()
```
:sparkles:

Reply:
or 
```
importOutboundStockMovement()
```

Reply:
I would go with the 2nd option, especially if we are supposed to distinguish the SMs in the future for inbound, outbound etc.